src/android/java/com/mattrayner/vuforia/app/ImageTargets.java

```diff
@@ -191,10 +217,36 @@ public void run() {
         }
     }
 
+
+    @Override
+    protected void onStart()
+    {
+        if (vuforiaActionReceiver == null)
+            vuforiaActionReceiver = new ActionReceiver();
+        IntentFilter intentFilter = new IntentFilter(VuforiaPlugin.PLUGIN_ACTION);
```

Comment:
Could you add braces around the `if` statement so it keeps in with the other `if` styling? Thanks